### PR TITLE
Uniformisation v2 / breaking changes

### PR DESCRIPTION
**Pull request by neolitec**
Base branch for all breaking changes about action-menus, checkboxes and spinners introduced by ui-kit v2.

### CODE REVIEW

File: client/app/ui-components/tile/tile.component.js [hunk 1]
```diff
@@ -135,34 +135,47 @@ angular.module("managerApp")
...
-                                <div class="oui-action-menu-item__icon"></div>
-                                <a class="oui-button oui-button_link oui-action-menu-item__label" data-ng-if="action.href"
+                <oui-dropdown class="cui-tile__item-button"
+                    data-align="end"
+                    arrow
+                    data-ng-if="$ctrl.hasAvailableAction()">
+                    <button
+                        oui-dropdown-trigger
+                        class="oui-action-menu-button">
+                        <svg xmlns="http://www.w3.org/2000/svg" viewbox="-1 -1 74 74">
```
Comment (Mboulianne) [inline client/app/ui-components/tile/tile.component.js]: do we really need to use a svh here?

File: client/app/ui-components/action-menu/action-menu.less [hunk 2]
```diff
@@ -1,11 +1,11 @@
 @import './_mixins';
 
-.oui-action-menu {
-  #oui > .oui-action-menu-base();
-}
+// .oui-action-menu {
```
Comment (Mboulianne) [inline client/app/ui-components/action-menu/action-menu.less]: Hmmm.... useless comments?

File: client/app/ui-components/tile/tile.component.js (hunk 1 above)
Comment (neolitec) [reply client/app/ui-components/tile/tile.component.js]: I saw an ui-sref a few lines below, so I can't use action-menu component without the  controller...
I chose to use oui-dropdown directly to handle ui-sref. I think it's more error-proof for now but yes, the drawback is that I have to insert an svg in the HTML :/

File: client/app/ui-components/action-menu/action-menu.less (hunk 2 above)
Comment (neolitec) [reply client/app/ui-components/action-menu/action-menu.less]: done

File: client/app/deskaas/deskaas.html [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <div ui-view class="cui-page">
-    
+
     <cui-page-header
         data-title-text="'vdi_header' | translate">
     </cui-page-header>
 
     <div class="cui-page__content">
-        <cui-loader data-ng-show="DeskaasCtrl.flags.initializing"></cui-loader>
+        <oui-spinner size data-ng-show="DeskaasCtrl.flags.initializing"></oui-spinner>
```
Comment (tremblaymath) [inline client/app/deskaas/deskaas.html]: size is orphan

File: client/app/iplb/frontends/iplb-frontends-edit.controller.js [hunk 4]
```diff
@@ -97,6 +97,7 @@ class IpLoadBalancerFrontendsEditCtrl {
                 this.type = "http";
                 this.frontend.port = 443;
                 this.frontend.ssl = true;
+                this.frontend.hsts = false;
```
Comment (tremblaymath) [inline client/app/iplb/frontends/iplb-frontends-edit.controller.js]: why is it in that pull request??

Comment (tremblaymath) [reply client/app/iplb/frontends/iplb-frontends-edit.controller.js]: nvm

File: client/app/iplb/home/iplb-home.html [hunk 5]
```diff
@@ -47,7 +47,12 @@ <h4 class="oui-header_4 cui-tile__title" data-translate="iplb_home_tile_usage"><
                             </div>
                         </div>
                         <div class="iplb-usage__menu">
-                            <cui-dropdown-menu>
+                            <oui-action-menu compact align="end">
+                                <oui-action-menu-item text="{{ 'iplb_utilisation_update_alert' | translate }}"
+                                    on-click="ctrl.updateQuotaAlert(quota)">
+                                </oui-action-menu-item>
+                            </oui-action-menu>
+                            <!-- <cui-dropdown-menu>
```
Comment (tremblaymath) [inline client/app/iplb/home/iplb-home.html]: dead code to remove

File: client/app/iplb/server/iplb-server-edit.html [hunk 6]
```diff
@@ -50,56 +50,35 @@ <h1 data-ng-hide="ctrl.edition" data-translate="iplb_server_add_title"></h1>
...
-                                <span class="oui-checkbox__label" data-translate="iplb_server_edit_ssl"></span>
-                                <span class="oui-checkbox__icon">
-                                    <i class="oui-icon oui-icon-checkbox-unchecked" aria-hidden="true"></i>
-                                    <i class="oui-icon oui-icon-checkbox-checked" aria-hidden="true"></i>
-                                    <i class="oui-icon oui-icon-checkbox-checkmark" aria-hidden="true"></i>
-                                </span>
-                            </label>
-                        </div>
+                        <oui-checkbox
+                            text="{{ 'iplb_server_edit_ssl' | translate }}"
```
Comment (tremblaymath) [inline client/app/iplb/server/iplb-server-edit.html]: All oui-checkbox text should be one-time binding

File: client/app/iplb/frontends/iplb-frontends-edit.controller.js (hunk 4 above)
Comment (neolitec) [reply client/app/iplb/frontends/iplb-frontends-edit.controller.js]: Indeterminate state on checkboxes has breaking changes too.

**Pull request merged**